app/models/service_retire_task.rb

```diff
@@ -28,8 +28,10 @@ def task_active
   def after_request_task_create
     update_attributes(:description => get_description)
     parent_svc = Service.find_by(:id => options[:src_ids])
-    _log.info("- creating service subtasks for service task <#{self.class.name}:#{id}>, service <#{parent_svc.id}>")
-    create_retire_subtasks(parent_svc, self)
+    unless retain_ansible_playbook_subtasks?(parent_svc)
```

Comment:
With the suggested change to the "create_subtasks?" method this would be easier to read as:
`if create_subtasks?(parent_svc)`